### PR TITLE
chatterino7: Add version 7.4.2

### LINKED ISSUE #10856: [Request]: Chatterino7
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Chatterino7

### Description

Chatterino7 is a fork of Chatterino with the implementation of 7tv

### Homepage

https://github.com/SevenTV/chatterino7

### Download Link(s)

https://github.com/SevenTV/chatterino7/releases

### Some Indication of Popularity/Repute

 246 stars at the moment
 https://7tv.app/

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #10856

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Zliced13** [comment]
WTF GITHUB ACTIONS IS SO STUPID.

**Comment by Zliced13** [comment]
@HUMORCE I think the GitHub actions bot is broken. My manifest is correct, but the bot thinks the manifest is incorrect.

File: bucket/chatterino7.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/SevenTV/chatterino7/releases/download/v$version/Chatterino7.Portable.zip"
+            }
+        },
+        "hash":{
+            "url": "$baseurl/sha256-checksums.txt",
+            "regex": "$sha256\\s+G\\:\\Dev\\symbolification\\final\\$basename"
```
Comment (StarsbySea) [inline bucket/chatterino7.json:41]:
```suggestion
            "url": "$baseurl/sha256-checksums.txt"
```

Comment (StarsbySea) [reply bucket/chatterino7.json:41]: Sorry, but there are some real problems with this manifest file haha, please see.
图片

**Review by StarsbySea — CHANGES_REQUESTED** [review]

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**